### PR TITLE
[JENKINS-36960] Change display name of AnyJobRestriction

### PR DESCRIPTION
**Pull request by alexsomai**
Relate to issue [[JENKINS-36960]](https://issues.jenkins-ci.org/browse/JENKINS-36960)

Summary of this pull request:
 - Change the display name of AnyJobRestriction to _No restriction (take any)_
 - Force AnyJobRestriction to be the first item in the list by using the `ordinal` method

CC @oleg-nenashev @martinda

### CODE REVIEW

File: src/main/java/com/synopsys/arc/jenkinsci/plugins/jobrestrictions/restrictions/logic/AnyJobRestriction.java [hunk 1]
```diff
@@ -49,8 +49,8 @@ public boolean canTake(Queue.BuildableItem item) {
     public boolean canTake(Run run) {
         return true;
     }
-        
-    @Extension
+
+    @Extension(ordinal = 1000)
```
Comment (alexsomai) [inline src/main/java/com/synopsys/arc/jenkinsci/plugins/jobrestrictions/restrictions/logic/AnyJobRestriction.java]: I think that it makes sense to display the `AnyJobRestriction` first item in the dropdown list.
screen shot 2016-07-27 at 5 20 24 pm

But @oleg-nenashev if you don't agree with this, I can revert it.

Comment (oleg-nenashev) [inline src/main/java/com/synopsys/arc/jenkinsci/plugins/jobrestrictions/restrictions/logic/AnyJobRestriction.java]: Makes sense

**Comment by oleg-nenashev** [comment]
👍

**Comment by martinda** [comment]
Nice and clear. Thanks.

**Pull request merged**